SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs

```diff
@@ -156,14 +156,16 @@ public static Domain.CaseSubmission ToDomain(this CaseSubmission caseSubmission)
                 PanelApprovedAt = caseSubmission.PanelApprovedAt,
                 PanelApprovedBy = caseSubmission.PanelApprovedBy?.ToDomain(false),
                 RejectionReason = caseSubmission.RejectionReason,
-                EditHistory = caseSubmission.EditHistory.Select(e => new EditHistory<Worker>
+                EditHistory = includeEditHistory ? caseSubmission.EditHistory.Select(e => new EditHistory<Worker>
                 {
                     EditTime = e.EditTime,
                     Worker = e.Worker.ToDomain(false)
-                }).ToList(),
+                }).ToList() : null,
                 SubmissionState = mapSubmissionStateToString[caseSubmission.SubmissionState],
-                FormAnswers = caseSubmission.FormAnswers,
-                Title = caseSubmission.Title
+                FormAnswers = includeFormAnswers ? caseSubmission.FormAnswers : null,
+                Title = caseSubmission.Title,
+                LastEdited = caseSubmission.EditHistory.Last().EditTime,
```

Comment:
Will initial submission have LastEdited value? So the .Last() will always have a value

Reply:
I'm tempted to make it nullable but based on FormUseCase#ExecutePost the initially created CaseSubmission does start with a single EditHistory item. .Last() should always return an item.